Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM alpine:3.4
+FROM alpine:latest
```

Comment:
while I love the `latest`, I also don't like things breaking unnecessarily.  Can you please peg a version here?